pkg/reconciler/managed/reconciler.go

```diff
@@ -506,6 +531,7 @@ type Reconciler struct {
 	log            logging.Logger
 	record         event.Recorder
 	metricRecorder MetricRecorder
+	changeLogger   ChangeLogger
```

Comment:
If you renamed `RecordChangeLog` to just `Log` and renamed this field to `change` then callers would write `r.change.Log(ctx, ...)`. I like the way that reads (more like prose, less stutter-ey).

Reply:
niiice, I like the way this reads too! good style here 😎 - did this in https://github.com/crossplane/crossplane-runtime/pull/754/commits/f7fb3a733a63a95ba0d4eb5f5555105d247dabd8